alf/utils/plot_tb_curves.py

```diff
@@ -643,12 +643,22 @@ def _get_curve_path(dir):
             smoothing=3) for t in tasks
     ] for m in methods]
 
+    # Specify y_range, or automatically find out y_range with margin
+    y_range = (0, 1.0)
+    _curves = [cr[0]() for cr in curve_readers]
+    if not y_range:
```

Comment:
This code block is just an example. perhaps it will be better to do this in the CurvesPlotter class?

Reply:
Realized that just specifying None will automatically determine y_range.  No need for this PR..